seed/views/v3/properties.py

```diff
@@ -1193,22 +1193,22 @@ def building_sync(self, request, pk):
         """
         Return BuildingSync representation of the property
         """
-        preset_pk = request.GET.get('preset_id')
```

Comment:
Still some hints of "preset" in this file that I think we can change now.

Reply:
Ah good find, I'll make that change